src/main/java/jung/gui/Jung.java

```diff
@@ -0,0 +1,98 @@
+package jung.gui;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import jung.exceptions.JungException;
+import jung.command.Command;
+import jung.parser.Parser;
+import jung.storage.Storage;
+import jung.storage.TaskList;
+
+
+public class Jung {
```

Comment:
The use of static fields and the initialized flag is a common pattern but can make the code harder to test and reason about. It's better to create an instance of Jung.
```suggestion
public class Jung {
    private TaskList taskList;
    private Storage storage;
    private boolean isInitialized = false;

    public Jung() {
        this.storage = new Storage("data/jung.txt");
    }

    public String initialize() {
        if (isInitialized) {
            return "";
        }
        try {
            this.taskList = new TaskList(storage.load(), storage);
        } catch (IOException e) {
            this.taskList = new TaskList(new ArrayList<>(), storage);
            isInitialized = true;
            return "Failed to load tasks, starting with an empty list.";
        }
        isInitialized = true;
        return getWelcomeMessage();
    }
    // ... rest of the methods become instance methods (remove 'static')
}
```